Some modifications, mainly to TapiPhotonicMedia:

amazzini opened the pull request:
- TapiConnectivity: ResilienceConstraint, waitToRevertTime, from simple integer (minutes) to more generic TimePeriod.
- TapiDigitalOtn: removed object key from codirectional of OduMip.
- TapiPhotonic: firstOfChain boolean added to Amplification.
- TapiPhotonic: added new PhotonicLayerQualifier OS_MEDIA (e.g. for Transponder line ports and ROADM add/drop ports).
- TapiPhotonic: comments enhanced.
- TapiPhotonic: OtsiaCsepTtpSpec renamed as OtsiaConnectivityServiceEndPointSpec, OtsiConfig as OtsiConfigPac.
- TapiPhotonic: PowerManagementConfigPac , intendedMaximumOutputPower to maxOutputPower and expected same.
- TapiPhotonic: PowerManagementCapabilityPac, maximum and minimum to max and min.
- TapiPhotonic: OtsiPowerConfig renamed as OtsiThresholdPowerConfig, deleted transmitPower and rxTotalPower, replaced by the composition of PowerManagementConfigPac.
- TapiPhotonic: added McGridConfigPac and OtsiMcGridConfigPac (N, M notation)

Commit:
Some modifications, mainly to TapiPhotonicMedia:

- TapiConnectivity: ResilienceConstraint, waitToRevertTime, from simple
integer (minutes) to more generic TimePeriod.
- TapiDigitalOtn: removed object key from codirectional of OduMip.
- TapiPhotonic: firstOfChain boolean added to Amplification.
- TapiPhotonic: added new PhotonicLayerQualifier OS_MEDIA (e.g. for
Transponder line ports and ROADM add/drop ports).
- TapiPhotonic: comments enhanced.
- TapiPhotonic: OtsiaCsepTtpSpec renamed as
OtsiaConnectivityServiceEndPointSpec, OtsiConfig as OtsiConfigPac.
- TapiPhotonic: PowerManagementConfigPac , intendedMaximumOutputPower to
maxOutputPower and expected same.
- TapiPhotonic: PowerManagementCapabilityPac, maximum and minimum to max
and min.
- TapiPhotonic: OtsiPowerConfig renamed as OtsiThresholdPowerConfig,
deleted transmitPower and rxTotalPower, replaced by the composition of
PowerManagementConfigPac.
- TapiPhotonic: added McGridConfigPac and OtsiMcGridConfigPac (N, M
notation)

Commit:
TapiPhotonic: added GridPac to OTS/OMS/MC/OTSiMC CEP.

Commit:
Other corrections to TapiPhotonic:

- Added “flexi” to the grid pacs.
- OtsiMcFrequencyConfigPac/centerFrequencyOffset is a signed int64, not
unit64 (before was of CentralFrequency type, removed as no longer used).
Same for N of FlexiGridConfigPac and FlexiGridPac.

Commit:
TapiStreaming: missing default values added to yang

nigel-r-davis reviewed (approved):
I have reviewed changes.

Commit:
Further corrections and modifications:

- TapiTopology: added RiskParameterPac to the Node.
- TapiDigitalOtn: minor correction to OtnOamServiceAugmentsOamService
augment.
- TapiConnectivity: ConnectivityObjectType, SERVER_CONSTRAINT corrected
as LAYER_PROTOCOL_CONSTRAINT.
- TapiPathComp: PathComputationObjectType,
PATH_COMPUTATION__SERVICE_END_POINT corrected as
PATH_COMP_PATH_SERVICE_END_POINT (shortened to "COMP" for 64 chars
limit)
- TapiOam: OamObjectType, PM_THRESHOLD_DATA corrected as PM_DATA.
- TapiCommon: created new Dc, Alr and Pm enums, to replace resp.
Detected, AlarmName and PmParameterName. Aligned to
“TAPI_Alarm_TCA_List_v2.0.0”.
- TapiFm: TcaInfo/ thresholdIndicatorName type from
TapiCommon:PmParamName to TapiCommon:Pm, AlarmInfo/ alarmName from
TapiCommon:AlarmName to TapiCommon:Alr, DetectedCondition/
detectedConditionName type from TapiCommon:Detected to TapiCommon:Dc.
- TapiOam: PmParameter/pmParameterName type from TapiCommon:PmParamName
to TapiCommon:Pm.
